src/__tests__/util/retry.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Stops execution for {@link milliseconds} milliseconds.
+ */
+export async function milliseconds(milliseconds: number): Promise<void> {
```

Comment:
Nit: can we explitily name this to include `wait`, for example?

Reply:
Sure! I've renamed it to `waitMilliseconds`, I'm open to suggestions though. Thanks!

[084c11b](https://github.com/safe-global/safe-client-gateway/pull/1437/commits/084c11b1fc1725728233c9ec9d376916e62a407c)